cli/cmd/install.go

```diff
@@ -122,15 +128,36 @@ const (
 	proxyInjectorTemplateName  = "templates/proxy_injector.yaml"
 )
 
-func newInstallOptions() *installOptions {
+func defaultInstallOptions() *installOptions {
```

Comment:
I'm ok with making this change, but all of our subcommands right now are structured to have a `<cmd>Options` struct and a `new<Cmd>Options` func that initializes the struct with default values. We should either update all of the `new<Cmd>Options` funcs in this package to instead be called `default<Cmd>Options`, or we should revert this rename in this branch and do a more comprehensive rename in a separate branch.

Reply:
Would it be better to call this newInstallOptionsWithDefaults?

Now that injetOptions/proxyConfigOptions do NOT come with defaults, it seems important to distinguish these.